src/test/java/com/google/api/codegen/go/GoContextCommonTest.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.go;
+
+import com.google.common.truth.Truth;
+import com.google.common.base.Joiner;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+/**
+ * Client config code generator baseline tests.
+ */
+public class GoContextCommonTest {
+
+  private static GoContextCommon ctx = new GoContextCommon();
+
+  @Test
+  public void testLineWrapShort() {
+    Truth.assertThat(ctx.getCommentLinesWrap("aaa")).containsExactly("// aaa");
+  }
+
+  @Test
+  public void testLineWrapLong() {
+    List<String> words = Arrays.asList(stringRep("a", 50), stringRep("a", 20), stringRep("a", 3));
```

Comment:
nit: alternate between "a", "b", and "c" to make it more even more obvious to humans these are distinct words. I know it doesn't matter to the correctness of the test.